cockroachcloud/run-bulk-operations.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Run Bulk Operations from Your Cluster
+summary: Run backups, restores, and imports from your CockroachCloud cluster.
+toc: true
+---
+
+The CockroachCloud tiers offer different levels of support for the following bulk operations. This page provides information on the availability of these operations in each CockroachCloud cluster tier and examples.
+
+- [`BACKUP`](../{{site.versions["stable"]}}/backup.html)
+- [`RESTORE`](../{{site.versions["stable"]}}/restore.html)
+- [`IMPORT`](../{{site.versions["stable"]}}/import.html)
+- [`EXPORT`](../{{site.versions["stable"]}}/export.html)
+- [`CREATE CHANGEFEED`](../{{site.versions["stable"]}}/create-changefeed.html)
+
+## Examples
+
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="cc-free">CockroachCloud Free (beta) </button>
```

Comment:
Might want to check that these `data-scope` filter names with dashes work with the JS in the build preview. I seem to remember something breaking before.